docs/content/commands/npm-audit.md

```diff
@@ -305,14 +305,24 @@ This value is not exported to the environment for child processes.
 
 #### `workspaces`
 
-* Default: false
-* Type: Boolean
+* Default: null
+* Type: null or Boolean
 
 Enable running a command in the context of **all** the configured
 workspaces.
 
+Explicitly setting this to false will cause commands like `install` to
+ignore workspaces altogether.
+
```

Comment:
The behavior of the default `null` value should be defined explicitly here.  Maybe something like this?

```
Set to true to run the command in the context of **all** configured workspaces.

Set to false to ignore workspaces altogether.

When not set explicitly:

- Commands that operate on the `node_modules` tree (install, update, etc.) will
   link workspaces into the `node_modules` folder.
- Commands that do other things (test, exec, publish, etc.) will operate on the root
  project, _unless_ one or more workspaces are specified in the `workspace` config.
```